content/docs/components-and-props.md

```diff
@@ -233,30 +234,30 @@ function Comment(props) {
 
 [](codepen://components-and-props/extracting-components-continued)
 
-Extracting components might seem like grunt work at first, but having a palette of reusable components pays off in larger apps. A good rule of thumb is that if a part of your UI is used several times (`Button`, `Panel`, `Avatar`), or is complex enough on its own (`App`, `FeedStory`, `Comment`), it is a good candidate to be a reusable component.
+Extraer componentes puede parecer un trabajo pesado al principio, pero tener una paleta de componentes reutilizables vale la pena en aplicaciones más grandes. Una buena regla en general es que si una parte de su UI se usa varias veces (`Button`, `Panel`, `Avatar`), o es lo suficientemente compleja por sí misma (`App`, `FeedStory`, `Comment`), es buen candidato para ser un componente reutilizable.
```

Comment:
```suggestion
Extraer componentes puede parecer un trabajo pesado al principio, pero tener una paleta de componentes reutilizables vale la pena en aplicaciones más grandes. Una buena regla en general es que si una parte de su UI se usa varias veces (`Button`, `Panel`, `Avatar`), o es lo suficientemente compleja por si misma (`App`, `FeedStory`, `Comment`), es buen candidato para ser un componente reutilizable.
```

Reply:
@alejandronanez I think it should be ticked

Reply:
Hey @EzequielMonforte, what do you mean by `ticketed`?

Reply:
You use `sí` if you're affirming something, `sí me gusta` for example.

Reply:
@alejandronanez  Actually @EzequielMonforte  is right. 😅
Check [DLE ](https://dle.rae.es/?id=XmOP8Ia|XmQKxT2)and [Fundéu](https://www.fundeu.es/recomendacion/en-si-mismo-concordancia-1576/)